### PR TITLE
Voting panel: execute + change vote

### PR DESCRIPTION
**Pull request by bpierre**
Related issue: https://github.com/aragon/aragon-apps/issues/216

### CODE REVIEW

File: apps/voting/app/src/components/VotePanelContent.js [hunk 1]
```diff
@@ -180,30 +205,82 @@ class VotePanelContent extends React.Component {
...
+                </ButtonsContainer>
+                <Info.Action>
+                  <p>
+                    You will cast {userBalance || '…'} votes, since it was your
+                    token balance at the{' '}
+                    <SafeLink
+                      href={`${etherscanBaseUrl}/block/${snapshotBlock}`}
+                      target="_blank"
+                    >
+                      beginning of the vote
```
Comment (izqui) [inline apps/voting/app/src/components/VotePanelContent.js]: I think we should print the block number here or add it on top right next to `quorum`, thoughts?

**Comment by izqui** [comment]
I'm also seeing some state propagation issues as in #448, after changing my vote the card view doesn't update the state.

2018-09-18 at 10 45 33 am

I'm testing using `aragon run` with a local chain.

File: apps/voting/app/src/components/VotePanelContent.js (hunk 1 above)
Comment (bpierre) [reply apps/voting/app/src/components/VotePanelContent.js]: Yes maybe we could have it at the top? @jounih 

In the meantime, I added it here.

**Comment by izqui** [comment]
@bpierre @jounih Replying to https://github.com/aragon/aragon-apps/pull/450#discussion_r218734782

👍. Should we also add it to the `You casted x votes` copy?

**Comment by bpierre** [comment]
@izqui yes, adding it

**Pull request merged**